Client/Frontend/Browser/URIFixup.swift

```diff
@@ -6,6 +6,26 @@ import Foundation
 import Shared
 
 class URIFixup {
+    private static func isValidIPAddressURL(_ string: String) -> Bool {
+        func isValidIPAddress(_ string: String) -> Bool {
+            var buffer = [UInt8](repeating: 0, count: Int(INET6_ADDRSTRLEN))
+            if inet_pton(AF_INET, string, &buffer) != 0 ||
+                inet_pton(AF_INET6, string, &buffer) != 0 {
+                return true
+            }
+            return false
+        }
+        
+        if let url = URL(string: "https://\(string)"),
+           let host = url.host, !host.isEmpty,
+           url.port != nil {
+            return isValidIPAddress(host)
+        } else if URL(string: "https://\(string)") != nil || URL(string: string) != nil {
+            return isValidIPAddress(string)
```

Comment:
`[::192.9.5.5]/ipng` fails the `isValidIPAddress` URL check - URL from the RFC: https://www.ietf.org/rfc/rfc2732.txt

Reply:
Fixed.